lib/shelley/src/Cardano/Wallet/Shelley.hs

```diff
@@ -288,7 +290,7 @@ serveWallet
                     Server.idleWorker
                 shelleyApi <- apiLayer (newTransactionLayer net) nl
                     (Server.manageRewardBalance proxy)
-                multisigApi <- apiLayer (newTransactionLayer net) nl
+                multisigApi <- apiLayer undefined nl
```

Comment:
I'm guessing there is some tricky type error?

Reply:
no no...there is required TxWintnessTag instance and I want to leave it for ADP-686. And for the current endpoints tx later is not engaged